week1/homework/src/server.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 const http = require('http');
-
-/* `createServer` MUST return an instance of `http.Server` otherwise the tests
- * will fail.
- */
 function createServer(port) {
   let state = 10;
 
   const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
+    switch (request.url) {
+      case '/state':
+        response.setHeader('Content-Type', 'application/json');
+        response.write(`{"state": ${state}}`);
+        break;
+      case '/add':
+        state++;
```

Comment:
Node that you kept app logic (adition) separate from other technical details (stringify) 👍